<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storey Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutBuilding">
     <item>
      <widget class="QLabel" name="buildingLabel">
       <property name="text">
        <string>Building:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBuilding"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Elevation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Height</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButton" stretch="0,0,0,0">
     <item>
      <widget class="QPushButton" name="buttonInsertAbove">
       <property name="text">
        <string>Insert Above</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonInsertBelow">
       <property name="text">
        <string>Insert Below</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMezzanine">
       <property name="text">
        <string>Mezzanine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
